### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by xdoo**

### CODE REVIEW

**Comment by xdoo** [comment]
see #24

**Comment by darkdragon-001** [comment]
@xdoo You can use [special keywords (like `fixes`)](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue) such that the linked issue will be closed automatically once the PR is merged :wink:

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
```
Comment (darkdragon-001) [inline .github/ISSUE_TEMPLATE/bug_report.md:4]: It often helps to prefix your issues.

```suggestion
title: '[bug] '
```

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: ''
```
Comment (darkdragon-001) [inline .github/ISSUE_TEMPLATE/feature_request.md:4]:
```suggestion
title: '[feature] '
```

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+labels: ''
```
Comment (darkdragon-001) [inline .github/ISSUE_TEMPLATE/feature_request.md:5]: Here, you could add your `enhancement` label:

```suggestion
labels: 'enhancement'
```

**Comment by xdoo** [comment]
@darkdragon-001 thanks for your suggestions. I think they make sense.

**Review by dominikhorn93 — APPROVED** [review]

**Pull request merged**